build-rirods.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
might rework this to be a Makefile itself...

targets could include...
- dockerimage
- devtoolsdocument
- devtoolsbuild